### PR TITLE
UI changes for Docker improvements

### PR DESCRIPTION
**Pull request by gartens**
These are the corresponding UI changes for [this pull request](https://github.com/polypheny/Polypheny-DB/pull/452).

### CODE REVIEW

File: src/app/components/toast/toast.model.ts [hunk 1]
```diff
@@ -25,12 +25,20 @@ export class Toast {
...
-        this.timeAsString = d.getHours() + ':' + d.getMinutes() + ':' + d.getSeconds();
+        this.timeAsString = this.twoDigits(d.getHours()) + ':' + this.twoDigits(d.getMinutes()) + ':' + this.twoDigits(d.getSeconds());
         this.type = type;
         this.delay = delay;//default 0 -> not removed automatically. if > 0: removed after n miliseconds
         this.hash = this.timeAsString + this.message;
     }
 
+    twoDigits(n: number): string {
+        let s = n.toString();
+        if (s.length == 1) {
```
Comment (datomo) [inline src/app/components/toast/toast.model.ts:36]: use "==="

File: src/app/views/schema-editing/graph-edit-graph/graph-edit-graph.component.ts [hunk 2]
```diff
@@ -145,15 +147,16 @@ export class GraphEditGraphComponent implements OnInit, OnDestroy {
...
-            }
+	    // This happens when we switch the namespace to a non-graph one on the sidebar,
+	    // because it takes time for the parent component to replace us with the correct one.
+            //if (this.dataPlacements.exception) {
+            //    // @ts-ignore
+            //    this._toast.exception({
+            //        error: this.dataPlacements.exception.detailMessage,
+            //        exception: this.dataPlacements.exception
+            //    });
+            //}
```
Comment (datomo) [inline src/app/views/schema-editing/graph-edit-graph/graph-edit-graph.component.ts:159]: Is this still required?

**Review by datomo — CHANGES_REQUESTED** [review]
Looks great, if the minor changes are done. Ok from my side.

File: src/app/views/schema-editing/graph-edit-graph/graph-edit-graph.component.ts (hunk 2 above)
Comment (gartens) [reply src/app/views/schema-editing/graph-edit-graph/graph-edit-graph.component.ts:159]: I just kept in, so it is clear why we are not displaying an error message.  We can also delete it or just keep the first two lines.

**Review by datomo — APPROVED** [review]
Good to go. 
Thanks @gartens  for the PR.

**Pull request merged**